.github/workflows/main.yml

```diff
@@ -0,0 +1,15 @@
+---
+name: Build
```

Comment:
Is the idea to turn this into a template? If so, should this become `repo2docker.yml` with `name: repo2docker` or similar?